### DISCUSSION TITLE
Configure and library support

### DISCUSSION THREAD

**Discussion by marcelchangTW** [MikePopoloski/slang#746] [General]
We have used slang to help us, and it's a great SV parser, thank Mike!

We have some cases which include configure and lib.map setting. Would like to know if you have any plan on this part? (33. Configuring the contents of a design)

**Comment by MikePopoloski** [comment]
Yes, I'm planning to eventually add support for all language features. I haven't looked too closely yet at the config features but now that I know somebody cares about it I can put it higher on the TODO list.

**Reply by jakelongo** [reply]
Have you had much headspace to think about how you'd prefer to go about adding support for the libraries and configs?

May have some time to contribute to slang but don't want to take it in the wrong direction if you've already got some ideas on how you'd like to have it implemented.

**Reply by MikePopoloski** [reply]
Yeah, this is next up on my TODO list and I have some initial pieces written already. I always welcome contributions but maybe pick a different area if you decide to do so.

**Reply by jakelongo** [reply]
Understood and thank you!
